tensorflow/contrib/factorization/python/ops/factorization_ops.py

```diff
@@ -745,3 +745,4 @@ def _process_input_helper(self, update_row_factors,
                                 update_indices,
                                 new_left_values,
                                 sharding_func))
+# pylint: enable=wildcard-import,undefined-variable
```

Comment:
Thanks for making these changes!  It would be nice to try to keep these re-enables as close to where they were disabled as possible.  So perhaps move this up near the imports, and preferably just after the last wildcard import?  

Same with some of the other files.